.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.4.1
\ No newline at end of file
+2.4.14
```

Comment:
I don't know if it's important but last time I was careful not to update the swagger version :)

Reply:
If everybody agrees to use the last version of `swagger-codegen@2` (2.4.14) then let's keep this update 👍

Reply:
I know, the generation have changed, so I noticed the generator was not the same, but you rolled back all the versions. I think we should keep the generated code and the mentioned version aligned